Phat-Taupe-Pests/BangazonAPI
# Eliza order products

## DESCRIPTION
Reformatted GET and POST on orders to be able to see the nested data on the single GET, and to post multiple products to an order.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In terminal, run `rm -rf Migrations bin obj bangazonapi.db`, `dotnet restore`, `dotnet ef migrations add {migrationName}`, `dotnet ef database update`, `dotnet build` then `dotnet run`
3. Testing requires entries in the orderProduct join table. POST the following to http://localhost:5000/order/addproduct 
	* `{"orderid" : 1 ,"productid" : 2}`
	* ` {"orderid" : 1 ,"productid" : 1} `
	* `{"orderid" : 1 ,"productid" : 1}`
4. GET : http://localhost:5000/order/1 Should return 2 tacos and a VW Beetle.
5. Format and values should match the format in the ticket

## REFERENCE

[Ticket](https://github.com/Phat-Taupe-Pests/BangazonAPI/issues/10)

[ERD](https://github.com/Phat-Taupe-Pests/Bangazon)
